website/docusaurus.config.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable global-require,import/no-extraneous-dependencies */
+const { config } = require('@apify/docs-theme');
+const { externalLinkProcessor } = require('./tools/utils/externalLink');
+const versions = require('./versions.json');
+
+const { absoluteUrl } = config;
+/** @type {Partial<import('@docusaurus/types').DocusaurusConfig>} */
+module.exports = {
+    title: 'Apify Docs v2',
```

Comment:
```suggestion
    title: 'Apify Documentation',
```

this will need to be fixed in all the projects, it is used in the title tag